inference-engine/src/vpu/common/include/vpu/ngraph/operations/out_shape_of_reshape.hpp

```diff
@@ -4,9 +4,7 @@
 
 #pragma once
 
-#include <ngraph/node.hpp>
-#include <ngraph/op/op.hpp>
-#include "ngraph/runtime/host_tensor.hpp"
+#include <ngraph/ngraph.hpp>
```

Comment:
@ilyachur it's still not clear for me - do we want to use global header for plugins as well? Or only for samples / template plugin

Reply:
I did in to cover all cases. Will revert back in order to don't affect compilation time.